docs/source/installation.rst

```diff
@@ -95,7 +93,6 @@ Windows
 3. Install GPkit
 ++++++++++++++++
   - Run ``pip install gpkit`` at an Anaconda Command Prompt.
-  - Run ``pip install pint`` for units support (recommended)

```

Comment:
I've updated the install test to check for the pint version, but it's commented out until you push to pipy.